.github/workflows/test_asterinas.yml

```diff
@@ -86,3 +86,56 @@ jobs:
       - name: General Test (Linux EFI Handover Boot Protocol)
         id: test_linux
         run: make run AUTO_TEST=test ENABLE_KVM=0 BOOT_PROTOCOL=linux-efi-handover64 RELEASE=1
+
+  integration-test-tdx:
```

Comment:
I think `integration-test` and `integration-test-tdx` should run in parallel to reduce our wait time for the Ci results to come out. Currently, `integration-test` is already the longest running test. If `integration-test-tdx` runs behind it, our CI test will be lengthened significantly.

Reply:
Currently they are tested in parallel because they are defined in different jobs.